test/passwordless/test_helpers_test.rb

```diff
@@ -46,7 +46,7 @@ class PasswordlessTestHelpersTest < ActiveSupport::TestCase
       assert 1, Session.count
       assert alice, Session.last!.authenticatable
       assert_match(
-        %r{^http://.*/users/sign_in/[a-z0-9]+/[a-z0-9]+}i,
+        %r{^http://.*/users/sign_in/[a-z0-9-]+/[A-Z0-9]+}i,
```

Comment:
With the `i` the case shouldn't matter, right?
```suggestion
        %r{^http://.*/users/sign_in/[a-z0-9-]+/[a-z0-9]+}i,
```